src/ragas/utils.py

```diff
@@ -22,6 +25,9 @@
 }
 # endpoint for uploading results
 RAGAS_API_URL = "https://api.ragas.io"
+RAGAS_APP_URL = "https://app.ragas.io"
+RAGAS_API_VERSION = "1.0.0"
```

Comment:
RAGAS_API_VERSION = "1.0.0"

app api version should be picked up from the internals. Not via env